app/controllers/api/v1/movies_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::MoviesController < ApplicationController
+  def index
+    movies = Movie.all
+    render json: movies 
+  end 
+
```

Comment:
Looks like this is also adding in feature related to an index page that should be on their own branch and not the `create_show_page`

Reply:
repulled after the index branch merge was approved, should be good i think